nodejs/week1/src/backend/routes/meal.js

```diff
@@ -0,0 +1,12 @@
+const mealsRouter = require('./meals.js');
+
+function mealRouter() {
+    let meals = mealsRouter.mealsRouter;
+    let item = meals[Math.floor(Math.random()*meals.length)];

```

Comment:
use const since you're not re-assigning them
it would also be nice to add some comments whenever youre doing funky stuff with math.floor/random... Someone else (like me in this case) has no idea why you're doing what you're doing :) Write code as if you'll have another person taking it over tomorrow and you think that he'll be judging you :D